### PR TITLE
init monorepo

### PR DESCRIPTION
**Pull request by soltanireza65**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (angelo-v) [inline .vscode/settings.json:1]: should this file be checked in?

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Samuele de Tomasi
```
Comment (angelo-v) [inline LICENSE.md:3]: Who is that?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "name": "root",
+  "private": true,
+  "workspaces": [
+    "demos/*",
+    "shared/*",
+    "packages/**/*"
+  ],
+  "scripts": {
+    "build": "npx lerna run build",
+    "publish": "npm run build && npx lerna publish --registry http://localhost:4873"
```
Comment (angelo-v) [inline package.json:11]: What is running on http://localhost:4873?

**Review by angelo-v — COMMENTED** [review]
I only reviewed the overall monorepo structure and did not take into account the contents of you packages / modules. If you want me to review those as well let me know, but this could be done independent of the monorepo change

File: package.json (hunk 3 above)
Comment (angelo-v) [reply package.json:11]: Please add a few words to the README about how this monorepo is used as a developer, e.g. about those scripts and the usage of lerna and nx

File: .vscode/settings.json (hunk 1 above)
Comment (soltanireza65) [reply .vscode/settings.json]: removed

**Comment by michielbdejong** [comment]
Reverted this PR due to outcome of conversation in #16

**Pull request merged**